numpy/lib/utils.py

```diff
@@ -193,7 +193,41 @@ def deprecate(*args, **kwargs):
     else:
         return _Deprecate(*args, **kwargs)
 
-deprecate_with_doc = lambda msg: _Deprecate(message=msg)
+
+def deprecate_with_doc(msg):
+    """
+    Decorator to deprecate a function and change its docstring to
+    mention the deprecation. 
+    
+    It returns an object that can be used to Issue a DeprecationWarning, 
+    by passing `func` as argument, this adds warning to the to-be 
+    decorated function's docstring and returns the new function object.
+    
+    See Also
+    --------
+    deprecate : Decorate a function such that it issues a `DeprecationWarning` 
+    
+    Parameters
+    ----------
+    msg : str
+    	    Additional explanation of the deprecation. Displayed in the 
+       	    docstring after the warning.
```

Comment:
indentation is off here

Reply:
Fixed.

Reply:
doesn't look fixed, `docstring` needs to line up with `Additional`

Reply:
The latest commits couldn't be pushed earlier. I believe this is fixed now.